README.md

```diff
@@ -17,9 +17,40 @@ A few key features:
 
 ## Getting Up and Running
 
-[Using a docker container](docs/docker_container.md)
+Start using OpenVino&trade; Model Server in 5 Minutes or less
```

Comment:
Change to `Start using OpenVINO Model Server in 5 Minutes or less:`

Reply:
done